content/en/platform/corda/5.2/deploying-operating/vnodes/bring-your-own-db.md

```diff
@@ -0,0 +1,144 @@
+---
+description: "Learn how to connect a virtual node to a database not managed by Corda."
+date: '2023-01-16'
+version: 'Corda 5.2'
+title: "Bringing Your Own Database"
+menu:
+  corda52:
+    parent: corda52-cluster-nodes
+    identifier: corda52-cluster-byod
+    weight: 900
+section_menu: corda52
+---
+
+# Bringing Your Own Database
+
+The default Corda deployment and migration functionality for [virtual nodes databases]({{< relref "../../key-concepts/cluster-admin/_index.md#virtual-node-databases" >}}) may not be suitable for environments with a controlled set of database permissions. Changes to database schemas may be a separate process run under the control of a Database Administrator. The bring-your-own-database (BYOD) feature enables Cluster Administrators to manage the creation and any subsequent updates of a PostgreSQL database, rather than using a database managed by Corda.
+
+{{< toc >}}
+
+## Creating Your Own Virtual Node Databases
+
+The following REST API endpoints are available to generate the SQL required to update the schemas required for a virtual node:
```

Comment:
I think we should delete one "required" from this sentence. Possibly the first one?